src/main/java/commands/Command.java

```diff
@@ -0,0 +1,32 @@
+package commands;
+
+import components.DukeException;
+import components.Storage;
+import components.Ui;
+import tasks.TaskList;
+
+public abstract class Command {
```

Comment:
Great work on creating a Command class and using it as a super class for all the commands supported.